src/transformers/models/tvlt/configuration_tvlt.py

```diff
@@ -0,0 +1,181 @@
+# coding=utf-8
+# Copyright 2022 MURGe-Lab and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" TVLT model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+TVLT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "TVLT/tvlt-base": "https://huggingface.co/TVLT/tvlt-base/blob/main/config.json",
+}
+
+
+class TvltConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`TvltModel`]. It is used to instantiate a TVLT
+    model according to the specified arguments, defining the model architecture. Instantiating a configuration with the
+    defaults will yield a similar configuration to that of the Tvlt
+    [TVLT/tvlt-base](https://huggingface.co/TVLT/tvlt-base) architecture.
+
+    Configuration objects inherit from [`PretrainedConfig`] and can be used to control the model outputs. Read the
+    documentation from [`PretrainedConfig`] for more information.
+
+    Args:
+        pixel_size (`int`, *optional*, defaults to 224):
+            The size (resolution) of each image.
+        audio_size (`int`, *optional*, defaults to 2048):
+            The time length of each audio spectrogram.
+        feature_size (`int`, *optional*, defaults to 128):
+            The frequency length of audio spectrogram.
+        pixel_patch_size (`int`, *optional*, defaults to 16):
```

Comment:
```suggestion
        image_patch_size (`int`, *optional*, defaults to 16):
```
Related to comment above